themes/book/app/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface TemplateOptions {
+  hide_toc?: boolean;
+  hide_outline?: boolean;
+  hide_footer_links?: boolean;
+  outline_maxdepth?: number;
+  hide_title_block?: boolean;
```

Comment:
Now that these are a nice clear type, it's easy to compare to the `template.yml` - and in doing so, I noticed `hide_title_block` is not declared there as an option. You mind adding that? (I see you already added a couple options to the article `template.yml`, so I assume it's easy.)